package.json

```diff
@@ -17,8 +17,7 @@
     "@smartthings/smartapp": "^1.0.3",
     "axios": "^0.19.2",
     "body-parser": "*",
-    "chai": "^4.2.0",
-    "chai-http": "^4.2.0",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-odetect-fixes",
```

Comment:
Reminder to update this version after https://github.com/bravetechnologycoop/brave-alert-lib/pull/3 is merged.